Framework/module_installer.py

```diff
@@ -1,21 +1,56 @@
+from curses import keyname
 import os
 import subprocess
 import sys
 import traceback
 
+from django import conf
```

Comment:
We cannot rely on django in node. @Muntasib-creator @Oytizzo verify and remove django from requirements.txt file.

Reply:
There is no django in requirements.txt. @Sakib75 remove this line because you didn't use this and what Sazid mentioned.

Reply:
Yes, the import was unintentional. I've removed the line of code